### ISSUE TITLE
What role does warmup.pth play and why is this model loaded during training

### ISSUE THREAD

**Issue by height921** [ericyinyzy/MTN_trajectory#4]
What role does warmup.pth play and why is this model loaded during training

**Comment by ericyinyzy** [comment]
That is a warm-up strategy, as illustrated in the training section of our paper. We first train an initialized transformer backbone for about ten epochs (warm-up.pth, only use bounding boxes ) and then add other modalities for the rest training process. Warm-up and training are conducted on the same datasets, the only difference is the input modalities. 

We find warm-up can be very useful for training a transformer on a small dataset. If you directly train different modalities together, the training loss decreases slowly, the model may even collapse.

**Comment by ericyinyzy** [comment]
That is a warm-up strategy, as illustrated in the training section of our paper. We first train an initialized single transformer backbone (without cross attention or optical flow stream) for about 10 epochs (warm-up.pth, only use bounding boxes ) and then add other modalities and model components for the rest training process. Warm-up and training are conducted on the same datasets, the only differences are input modalities and model structure.

We find warm-up can be very useful for training a transformer on a small dataset. If you directly train different modalities together, the training loss decreases slowly, the model may even collapse.

**Comment by height921** [comment]
Thank you for your reply. I understand. Thank you again
Thank you for your reply. I understand. Thank you again

同学
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ericyinyzy/MTN_trajectory"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月25日(星期二) 上午9:55
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ericyinyzy/MTN_trajectory] What role does warmup.pth play and why is this model loaded during training (Issue #4)





 
That is a warm-up strategy, as illustrated in the training section of our paper. We first train an initialized single transformer backbone (without cross attention or optical flow stream) for about 10 epochs (warm-up.pth, only use bounding boxes ) and then add other modalities and model components for the rest training process. Warm-up and training are conducted on the same datasets, the only differences are input modalities and model structure.
 
We find warm-up can be very useful for training a transformer on a small dataset. If you directly train different modalities together, the training loss decreases slowly, the model may even collapse.
